compiler/rustc_codegen_llvm/src/context.rs

```diff
@@ -21,7 +21,9 @@ use rustc_middle::ty::layout::{
 };
 use rustc_middle::ty::{self, Instance, Ty, TyCtxt};
 use rustc_middle::{bug, span_bug};
-use rustc_session::config::{BranchProtection, CFGuard, CrateType, DebugInfo, PAuthKey, PacRet};
+use rustc_session::config::{
+    BranchProtection, CFGuard, CFProtection, CrateType, DebugInfo, PAuthKey, PacRet,
+};
```

Comment:
Nit: could you avoid multi-line imports?

Reply:
Sure, but how? `./x.py fmt` fixes things up this way and I notice that the same file has multi-line imports above and below this line.

Reply:
Split up the import into multiple.

e.g.

```
use rustc_session::config::{A, B, C, X, Y};
use rustc_session::config::Z;